site/content/blog/2025/12/24/aws-kms.md

```diff
@@ -0,0 +1,98 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+title: "Securing S3 data with AWS KMS"
+date: 2025-12-24
+author: Dmitri Bourlatchkov
+---
+## Introduction
+
+AWS [Key Management Service](https://docs.aws.amazon.com/kms/latest/developerguide/overview.html) (KMS) provides
+a way to encrypt S3 data in AWS without exposing raw key material outside AWS services.
+
+Apache Polaris supports using KMS in its catalogs backed by AWS S3 storage.
+
+The core functionality is available via Polaris REST API since the `1.2.0-incubating` release.
+CLI support will be made available in the release following `1.3.0-incubating`.
```

Comment:
Are we planning to CP https://github.com/apache/polaris/pull/3330 this tp 1.3x ? and have a new RC ?

Reply:
I guess not. Let's not expand scope of 1.3.0

Reply:
I was trying to understand more what the above statement meant, 1.3.1 could also be a following release. 
i think we mean 1.4 ideally and we are not sure if will be 2.0 ?